ecosystem/typescript/sdk/src/tests/e2e/pagination_helper.test.ts

```diff
@@ -0,0 +1,25 @@
+import { get } from "../../client";
+import * as Gen from "../../generated";
+import { paginateWithCursor } from "../../utils/pagination_helpers";
+import { longTestTimeout } from "../unit/test_helper.test";
+
+test(
+  "gets the full data with pagination",
+  async () => {
+    try {
+      const modules = await get<{}, Gen.MoveModuleBytecode[]>({
+        url: "https://fullnode.testnet.aptoslabs.com/v1",
+        endpoint: "accounts/0x1/modules",
+      });
+      const paginateOut = await paginateWithCursor<{}, Gen.MoveModuleBytecode[]>({
+        url: "https://fullnode.testnet.aptoslabs.com/v1",
+        endpoint: "accounts/0x1/modules",
+        params: { limit: 20 },
+      });
```

Comment:
https://fullnode.testnet.aptoslabs.com/v1/accounts/0x1/modules has 83 modules. This tests shows that the `paginateWithCursor` function works and eventually returns all 83 modules event though we have the limit set to 20. 
Isn't that how the function suppose to work? am I missing something?

Reply:
So this first part:
```
      const modules = await get<{}, Gen.MoveModuleBytecode[]>({
        url: "https://fullnode.testnet.aptoslabs.com/v1",
        endpoint: "accounts/0x1/modules",
      });
```

If there are more than PAGE_SIZE modules this will only return PAGE_SIZE modules and this test will fail.

Reply:
Okay we just spoke offline. The configured default max page size for modules is 9999 so realistically this won't be a problem. @0xmaayan if you change the first call to include a page size of 9999, and include a comment explaining that if 0x1 ever has more than 9999 modules this test will fail, then I think we're good to go.